app/models/dialog.rb

```diff
@@ -61,6 +61,10 @@ def validate_children
       errors.add(:base, _("Dialog %{dialog_label} must have at least one Tab") % {:dialog_label => label})
     end
 
+    if dialog_fields.collect(&:name).duplicates.present?
```

Comment:
@d-m-u Set `dialog_fields.collect(&:name).duplicates` to a variable so we do not need to call it twice if there are duplicates.